### PR TITLE
fix(tests): remove md5sum for all mosdepth tests + update dependency for these tests

### PR DESCRIPTION
**Pull request by maxulysse**
Should fix all failing tests in the release PR

## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/sarek/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/sarek _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nf-test test tests/ --verbose --profile +docker`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: tests/test_gatk4spark.yml [hunk 1]
```diff
@@ -78,7 +69,6 @@
     - path: results/reports/markduplicates
       should_exist: false
     - path: results/reports/mosdepth
```
Comment (maxulysse) [inline tests/test_gatk4spark.yml:71]:
```suggestion
    - path: results/reports/mosdepth
      should_exist: false
```

File: tests/test_prepare_recalibration_from_bam.yml [hunk 2]
```diff
@@ -75,7 +70,6 @@
     - path: results/preprocessing/recalibrated/test/test.recal.cram.crai
       should_exist: false
     - path: results/reports/mosdepth
```
Comment (maxulysse) [inline tests/test_prepare_recalibration_from_bam.yml:72]:
```suggestion
    - path: results/reports/mosdepth
      should_exist: false
```

File: tests/test_prepare_recalibration_from_cram.yml [hunk 3]
```diff
@@ -69,13 +64,9 @@
     - path: results/preprocessing/recalibrated/test/test.recal.cram.crai
       should_exist: false
     - path: results/reports/mosdepth/test/test.recal.mosdepth.global.dist.txt
```
Comment (maxulysse) [inline tests/test_prepare_recalibration_from_cram.yml:66]:
```suggestion
    - path: results/reports/mosdepth/test/test.recal.mosdepth.global.dist.txt
      should_exist: false
``````

File: tests/test_prepare_recalibration_from_cram.yml [hunk 4]
```diff
@@ -69,13 +64,9 @@
...
-      should_exist: false
     - path: results/reports/mosdepth/test/test.recal.mosdepth.region.dist.txt
```
Comment (maxulysse) [inline tests/test_prepare_recalibration_from_cram.yml:67]:
```suggestion
    - path: results/reports/mosdepth/test/test.recal.mosdepth.region.dist.txt
      should_exist: false
``````

File: tests/test_prepare_recalibration_from_cram.yml [hunk 5]
```diff
@@ -69,13 +64,9 @@
...
-      should_exist: false
     - path: results/reports/mosdepth/test/test.recal.mosdepth.summary.txt
```
Comment (maxulysse) [inline tests/test_prepare_recalibration_from_cram.yml:68]:
```suggestion
    - path: results/reports/mosdepth/test/test.recal.mosdepth.summary.txt
      should_exist: false
``````

File: tests/test_prepare_recalibration_from_cram.yml [hunk 6]
```diff
@@ -69,13 +64,9 @@
...
-      should_exist: false
     - path: results/reports/mosdepth/test/test.recal.regions.bed.gz
```
Comment (maxulysse) [inline tests/test_prepare_recalibration_from_cram.yml:69]:
```suggestion
    - path: results/reports/mosdepth/test/test.recal.regions.bed.gz
      should_exist: false
``````

File: tests/test_recalibrate_from_bam.yml [hunk 7]
```diff
@@ -72,13 +67,9 @@
     - path: results/preprocessing/recalibrated/test/test.recal.cram.crai
       should_exist: false
     - path: results/reports/mosdepth/test/test.recal.mosdepth.global.dist.txt
```
Comment (maxulysse) [inline tests/test_recalibrate_from_bam.yml:69]:
```suggestion
    - path: results/reports/mosdepth/test/test.recal.mosdepth.global.dist.txt
      should_exist: false
``````

File: tests/test_recalibrate_from_bam.yml [hunk 8]
```diff
@@ -72,13 +67,9 @@
...
-      should_exist: false
     - path: results/reports/mosdepth/test/test.recal.mosdepth.region.dist.txt
```
Comment (maxulysse) [inline tests/test_recalibrate_from_bam.yml:70]:
```suggestion
    - path: results/reports/mosdepth/test/test.recal.mosdepth.region.dist.txt
      should_exist: false
``````

File: tests/test_recalibrate_from_bam.yml [hunk 9]
```diff
@@ -72,13 +67,9 @@
...
-      should_exist: false
     - path: results/reports/mosdepth/test/test.recal.mosdepth.summary.txt
```
Comment (maxulysse) [inline tests/test_recalibrate_from_bam.yml:71]:
```suggestion
    - path: results/reports/mosdepth/test/test.recal.mosdepth.summary.txt
      should_exist: false
``````

File: tests/test_recalibrate_from_bam.yml [hunk 10]
```diff
@@ -72,13 +67,9 @@
...
-      should_exist: false
     - path: results/reports/mosdepth/test/test.recal.regions.bed.gz
```
Comment (maxulysse) [inline tests/test_recalibrate_from_bam.yml:72]:
```suggestion
    - path: results/reports/mosdepth/test/test.recal.regions.bed.gz
      should_exist: false
``````

File: tests/test_recalibrate_from_cram.yml [hunk 11]
```diff
@@ -68,13 +63,9 @@
     - path: results/preprocessing/recalibrated/test/test.recal.cram.crai
       should_exist: false
     - path: results/reports/mosdepth/test/test.recal.mosdepth.global.dist.txt
```
Comment (maxulysse) [inline tests/test_recalibrate_from_cram.yml:65]:
```suggestion
    - path: results/reports/mosdepth/test/test.recal.mosdepth.global.dist.txt
      should_exist: false
``````

File: tests/test_recalibrate_from_cram.yml [hunk 12]
```diff
@@ -68,13 +63,9 @@
...
-      should_exist: false
     - path: results/reports/mosdepth/test/test.recal.mosdepth.region.dist.txt
```
Comment (maxulysse) [inline tests/test_recalibrate_from_cram.yml:66]:
```suggestion
    - path: results/reports/mosdepth/test/test.recal.mosdepth.region.dist.txt
      should_exist: false
``````

File: tests/test_recalibrate_from_cram.yml [hunk 13]
```diff
@@ -68,13 +63,9 @@
...
-      should_exist: false
     - path: results/reports/mosdepth/test/test.recal.mosdepth.summary.txt
```
Comment (maxulysse) [inline tests/test_recalibrate_from_cram.yml:67]:
```suggestion
    - path: results/reports/mosdepth/test/test.recal.mosdepth.summary.txt
      should_exist: false
``````

File: tests/test_recalibrate_from_cram.yml [hunk 14]
```diff
@@ -68,13 +63,9 @@
...
-      should_exist: false
     - path: results/reports/mosdepth/test/test.recal.regions.bed.gz
```
Comment (maxulysse) [inline tests/test_recalibrate_from_cram.yml:68]:
```suggestion
    - path: results/reports/mosdepth/test/test.recal.regions.bed.gz
      should_exist: false
``````

**Comment by asp8200** [comment]
Were those tests failing because of my update of mosdepth?  If so, why didn't those tests failing on my [PR](https://github.com/nf-core/sarek/pull/1490)?  Weren't those tests triggered on my PR?

**Comment by asp8200** [comment]
> Were those tests failing because of my update of mosdepth? If so, why didn't those tests failing on my [PR](https://github.com/nf-core/sarek/pull/1490)? Weren't those tests triggered on my PR?

Ah, I see that some dependencies to mosdepth were missing in `tests/config/pytesttags.yml`. Thanks for fixing that 👏

**Review by asp8200 — APPROVED** [review]

**Pull request merged**